.gitignore

```diff
@@ -6,3 +6,4 @@ messages.pot
 nbproject
 .metadata
 /.env
+package-lock.json
```

Comment:
???

Reply:
> 
> 
> Why is the yarn.lock deleted in the backend folder? What does this new package?

Sorry, I have forget to restart the container to get a rebuilded yarn.lock.

Reply:
I use npm because I have problems with yarn and to prevent problems with yarn for the other, I have added my package-lock.json to gitignore